airbyte-webapp/src/pages/OnboardingPage/useStepsConfig.tsx

```diff
@@ -13,7 +14,14 @@ const useStepsConfig = (
   setCurrentStep: (step: StepType) => void;
   steps: Array<{ name: JSX.Element; id: StepType }>;
 } => {
+  // exp-speedy-connection
+  const location = useLocation() as unknown as ILocationState<{ step: StepType }>;
```

Comment:
:broom: We'd usually encapsulate this into a separate hook `useLocationState` that just takes a generic, so we don't need to cast in the consuming place to unknown and back, but have that ugly cast limitd to the one place inside the hook.